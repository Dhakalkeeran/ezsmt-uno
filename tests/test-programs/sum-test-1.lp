#theory lp {
    linear_term {
    - : 2, unary;
    * : 1, binary, left;
    + : 0, binary, left;
    - : 0, binary, left
    };

    dom_term {
    - : 3, unary;
    + : 3, unary;
    * : 2, binary, left;
    + : 1, binary, left;
    - : 1, binary, left;
    .. : 0, binary, left
    };

    &sum/0   : linear_term, {<=,>=,>,<,=,!=}, linear_term, any;
    &logic/1 : linear_term, head
}.

&logic(lia).
node(6).
&sum {4; X; 10} = at(X) :- node(X).
