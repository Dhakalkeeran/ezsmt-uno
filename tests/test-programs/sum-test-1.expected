node(6) 
at(6)=20
